### ISSUE TITLE
ArrowParseWarning

### ISSUE THREAD

**Issue by TomStageDK** [csirtgadgets/bearded-avenger#476]
Hi All,

Having done a fresh installation today on a Ubuntu 16.04 (fully updated system), waited for a while to make sure that data would be in the DB (default sqlite), ran a simple query and gotr the below error, looking at the referenced github page given in the error they say that the error can be supressed, so my question is how / where should the ArrowParseWarning be suppressed?

Query:
cif --itype ipv4 --limit 5 --format table --tags scanner

Error / Warning:
/usr/local/lib/python2.7/dist-packages/arrow/factory.py:202: ArrowParseWarning: The .get() parsing method without a format string will parse more strictly in version 0.15.0.See https://github.com/crsmithdev/arrow/issues/612 for more details.
  ArrowParseWarning,

If more data is needed let me know.
Cheers,
Tom

**Comment by JesseBowling** [comment]
I also encountered this yesterday for the first time. I suspect use of arrow in this repo should be updated....

**Comment by JesseBowling** [comment]
Looks like 6 references:

https://github.com/csirtgadgets/bearded-avenger/search?q=%22arrow.get%22&unscoped_q=%22arrow.get%22

**Comment by JesseBowling** [comment]
If disabling warnings:
```
Disable Warnings

To get rid of the ArrowParseWarning messages in 0.14.3 onwards, do the following:

import warnings
from arrow.factory import ArrowParseWarning

warnings.simplefilter("ignore", ArrowParseWarning)
```

also might be addressed with this + freezing the version of arrow. If code is going to be fixed and a format string must be provided, that will likely mean additional specification in docs/etc. :(

**Comment by wesyoung** [comment]
I’d just submit a PR that has a pinned version of the previous version of arrow w/o that warning... I did the same in verbose-robot today.. (v4)

**Comment by systemcatch** [comment]
Hey everyone, I'm one of the maintainers of arrow.

arrow 0.14.3+ have the ArrowParseWarning included. arrow has long standing bugs related to str parsing that we're attempting to fix in version 0.15.0 and some of these changes may cause breakages.

Looking at the 6 references provided by @JesseBowling I don't see anything that problematic.

- If you can tell me the format expected for things like reporttime, lasttime, 'time_first' etc I can help with any problems, since this package seems to deal with sqlite you're likely ok.

- There will need to be a change from;

```python
if isinstance(timestamp, str):
            timestamp = arrow.get(timestamp).datetime
```

to

```python
if isinstance(timestamp, str):
            timestamp = arrow.get(timestamp, "X").datetime
```

Version 0.15.0 will be released near the end of August, the warnings will be gone then.
